server/prisma/schema.prisma

```diff
@@ -39,6 +39,9 @@ model DonatedItem {
   donor         Donor               @relation(fields: [donorId], references: [id])
   program       Program             @relation(fields: [programId], references: [id])
   statuses      DonatedItemStatus[]  // Relation to DonatedItemStatus
+
+  imageUrls     String[]            // Array of image URLs or base64 strings
+
```

Comment:
@miabu-pashh revert changes in this file, actually all changes in `/server` folder. The issue assigned to you in Sprint 3 is related to frontend. You need not to do any changes in backend. We are not going to store images directly in postgres. We will be using Azure cloud for that, this is not the process we want to follow, So please revert all files from` /server `directory